SDPi_Supplement/asciidoc/volume2/gateways/tf2-ch-b-gateway-private-mdc-mapping.adoc

```diff
@@ -0,0 +1,66 @@
+[#ref_private_coding_system_note]
+==== Private MDC Codes Consideration
+The default coding system utilized in SDC is *"MDC"* (Medical Device Communications) also known as *"ISO/IEEE 11073-10101"* nomenclature.
+
+In addition to the standard codes, *MDC* defines private code ranges that can be used by medical device vendors for defining their own codes, for example, for concepts for which standard codes do not exist. Private codes should be avoided whenever possible since this undermines interoperability.
+
+Coded elements in SDC using *MDC* private codes shall also contain a *pm:Translation* element that defines the vendor-specific coding system.

```

Comment:
This can also be linked to bpkp:TR1358:

`If an SDC BASE PARTICIPANT uses a private code as detailed in [IEEE 11073-10101], the SDC BASE PARTICIPANT SHALL refer to its definition by providing a pm:Translation with a @CodingSystem that uniquely identifies the private coding system that contains the code.`

Reply:
Added a reference to the requirement in the B-PKP.